torch/distributed/_sharded_tensor/__init__.py

```diff
@@ -112,6 +162,109 @@ def ones(
         init_rrefs=init_rrefs,
     )
 
+
+def zeros(sharding_spec: ShardingSpec,
+          *size,
+          dtype=None,
+          layout=torch.strided,
+          requires_grad=False,
+          pin_memory=False,
+          memory_format=torch.contiguous_format,
+          process_group=None,
+          init_rrefs=False):
+    """
+    Creates a zeros :class:`ShardedTensor`. Needs to be called on all ranks in an SPMD fashion.
```

Comment:
Can use the docs from `torch.zeros`: `Returns a tensor filled with the scalar value 0`. For ShardedTensor it could be something like:

```
Returns a :class:`ShardedTensor` filled with the scalar value 0. Needs to be called on all ranks in an SPMD fashion.
```

Reply:
Done